src/utils/generateJailsKeyInsightsData.js

```diff
@@ -0,0 +1,139 @@
+// Recidiviz - a data platform for criminal justice reform
+// Copyright (C) 2020 Recidiviz, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+// =============================================================================
+import {
+  POPULATION_JAIL,
+  PERCENTAGE_COVERED_COUNTY,
+  PERCENTAGE_COVERED_POPULATION,
+  INCARCERATION_RATE_JAIL,
+  metricToCardName,
+} from "../constants/metrics";
+import formatNumber from "./formatNumber";
+
+/**
+ * @param data - normalized, grouped and sorted metric data (output of `getNormalizedStateData`)
```

Comment:
What is this docstring attached to? Is it supposed to be attached to `generateJailsKeyInsightsData` below?

Reply:
Yes, it is for `generateJailsKeyInsightsData`. Moved it below.